runtime/src/bank/builtins/mod.rs

```diff
@@ -323,10 +316,6 @@ mod tests {
             &super::BUILTINS[2].core_bpf_migration_config,
             &Some(super::test_only::stake_program::CONFIG)
         );
-        assert_eq!(
-            &super::BUILTINS[3].core_bpf_migration_config,
-            &Some(super::test_only::config_program::CONFIG)
-        );
```

Comment:
nit: Can you just add a comment here saying that config isn't tested because it's not in test_only mode anymore?

Reply:
Hah, I was so close to adding a comment for this. Smh.